README.md

```diff
@@ -152,11 +152,15 @@ Gatekeeper requires some configuration for it to run in your environment, see be
 These parameters are used by both the EC2 and RDS Gatekeeeper services 
 
 #### Authorization
-Currently Gatekeeper only supports authorization through LDAP, the application expects authentication to be done via SSO, and looks for the username in a header.
+Currently Gatekeeper supports authorization through LDAP and Header Injection, the application expects authentication to be done via SSO, and looks for the username in a header.
 
```

Comment:
We might want to call out in the documentation on what a "header" should look like if there exists one out there who wants to use this feature.'

the examples shouldn't reference internal things so be mindful of that when you are making your updates..